pandas/tests/frame/test_alter_axes.py

```diff
@@ -317,12 +317,17 @@ def test_convert_dti_to_series(self):
         tm.assert_series_equal(result, expected)
 
         # convert to utc
-        df['B'] = idx.to_series(index=[0, 1])
+        df['B'] = idx.to_series(keep_tz=False, index=[0, 1])
         result = df['B']
         comp = Series(DatetimeIndex(expected.values).tz_localize(None),
                       name='B')
         tm.assert_series_equal(result, comp)
 
+        with tm.assert_produces_warning(FutureWarning):
+            result = idx.to_series(index=[0, 1])
+        tm.assert_series_equal(
+            result, expected.dt.tz_convert('UTC').dt.tz_localize(None))
```

Comment:
This might seem a strange file to put this test, but as far as I found, this is actually the only place where this behaviour was tested. So I decided to put it here, so only one test needs to be updated when we remove the deprecation.